rollup/internal/controller/relayer/l1_relayer.go

```diff
@@ -150,6 +154,15 @@ func (r *Layer1Relayer) ProcessGasPriceOracle() {
 		}
 
 		if r.shouldUpdateGasOracle(baseFee, blobBaseFee, isCurie) {
+			// if we are not committing batches due to high fees then we shouldn't update fees to prevent users from paying high l1_data_fee
```

Comment:
```suggestion
			// It indicates the commit/finalize batch has been stuck for a long time, it's likely that the L1 gas fee spiked.
			// If we are not committing batches due to high fees then we shouldn't update fees to prevent users from paying high l1_data_fee
```